lisa/sut_orchestrator/baremetal/cluster/cluster.py

```diff
@@ -93,5 +93,14 @@ def _fill_capability(self, node_capability: schema.NodeSpace) -> None:
             items=[
                 schema.FeatureSettings.create(features.SerialConsole.name()),
                 schema.FeatureSettings.create(features.StartStop.name()),
+                schema.FeatureSettings.create(features.SecurityProfile.name()),
             ],
         )
+
+        # Convert generic FeatureSettings to baremetal-specific ones
+        from ..platform_ import BareMetalPlatform
```

Comment:
Why does it need to reload here? What's different the SecurityProfile with SerialConsole or StartStop?